app/controllers/invite_controller.rb

```diff
@@ -82,16 +83,16 @@ def submit
         @type = "danger"
       else
         tester = Spaceship::Tunes::Tester::External.new({
-          'emailAddress' => {'value' => email},
```

Comment:
Why did this API change? I think it's because we shouldn't call this method, but some other one?